src/AddTodoForm.js

```diff
@@ -1,20 +1,31 @@
 import React from "react";
+import { useState } from "react";
 
-function AddTodoForm(props){
-   const  handleAddTodo =(event)=>{
+function AddTodoForm({onAddTodo}) {
+    const [todoTitle, setTodoTitle] = useState('');
+
+  const handleTitleChange = (e) => {
+    const newTodoTitle = e.target.value;
+    setTodoTitle(newTodoTitle);
+  };
+  const handleAddTodo = (event) => {
     event.preventDefault();
-    const todoTitle = event.target.title.value; 
     console.log(todoTitle);
-    props.onAddTodo(todoTitle);
-
```

Comment:
after you are done with adding todoTitle to your list you can reset a todoTitle value to an empty string
setTodoTitle('')